test/unit/sidekiq_logging_middleware_test.rb

```diff
@@ -17,4 +17,19 @@ class SidekiqLoggingMiddlewareTest < ActiveSupport::TestCase
 
     middleware.call('DummyWorker', msg) { nil }
   end
+
+  test 'do not filter plain arrays arguments' do
```

Comment:
What is this test for?

Reply:
It was added in this commit: https://github.com/3scale/porta/pull/3595/commits/fd57f41b5a62869c762dbd7f558563cbb161d2fc

I'll later refactor the PR, to make sure that this commit is visible.

With the original implementation of filtering, all of the array arguments of the jobs were filtered, so the log for the test would be something like:
```
'Enqueued DummyWorker#123 with args: ["some_arg", [nil, nil, nil, nil, nil, nil]]'
```